pkg/sentry/metrics.go

```diff
@@ -0,0 +1,83 @@
+package sentry
+
+import (
+	"fmt"
+	"net/url"
+	"strconv"
+	"time"
+)
+
+type MetricsResponse struct {
+	Start     time.Time   `json:"start"`
+	End       time.Time   `json:"end"`
+	Intervals []time.Time `json:"intervals"`
+	Groups    []struct {
+		By     map[string]interface{} `json:"by"`
+		Totals map[string]interface{} `json:"totals"`
+		Series map[string]interface{} `json:"series"`
+	} `json:"groups"`
+}
+
+type GetMetricsInput struct {
+	OrganizationSlug string
+	ProjectIds       []string
+	Environments     []string
+	From             time.Time
+	To               time.Time
+	Interval         time.Duration
+	Field            string
+	Query            string
+	GroupBy          string
+	Sort             string
+	Order            string
+	Limit            int64
+}
+
+func (args *GetMetricsInput) ToQuery() string {
```

Comment:
Same as above, could this function be broken down a little more?

Reply:
Same here, the function is pretty simple and doesn't really have any parts that benefit extracting IMHO and is no different from all the other `ToQuery()` functions.